File: Data Structures & Algorithms/rotate-array/submission-3.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        n=len(nums)
        nums1=[0]*n
        k=k%n
        for i in range(len(nums)):
            nums1[(i+k)%n]=nums[i]
        for i in range(len(nums)):
            nums[i]=nums1[i]
        

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-3.java
class Solution {
    public int minSubArrayLen(int target, int[] nums) {
        //Brute Force
        int minimum=Integer.MAX_VALUE;
        int length=0;
        for(int i=0;i<nums.length;i++){
            int sum=0;
            for(int j=i;j<nums.length;j++){
                sum+=nums[j];
                if(sum>=target){
                    length=j-i+1;
                    minimum=Math.min(minimum,length);
                }
            }
        }
        return minimum==Integer.MAX_VALUE?0:minimum;
    }
}